app/services/scenes-collections/nodes/sources.ts

```diff
@@ -1,12 +1,14 @@
 import { Node } from './node';
 import { SourcesService, Source, TSourceType } from '../../sources';
 import { FiltersNode } from './filters';
+import { FontLibraryService } from '../../font-library';
```

Comment:
We've started importing things relative to `app` rather than relative to the file it's in.  This makes things very easy when moving files around.  So for example you can do this import as `import { FontLibraryService } from 'services/font-library';`.  We have webpack set up so it will recognize imports relative to the `app` directory.  The only exception to this is when two files are very tightly coupled and will always be in the same place (part of the same service).  In that case it's fine to do a relative import.

Reply:
I changed this and for consistency, changed the others. See if that looks alright.